addon/services/unified-event-handler.js

```diff
@@ -106,7 +106,8 @@ export default Ember.Service.extend(Ember.Evented, {
     if (!handlerInfo) {
       // Add new DOM event listener since there is none
       let emberEventName = `${eventName}.${generateId()}`;
-      let trigger = this.triggerEvent.bind(this, emberEventName);
+      const throttledEventCallback = this._generateEventThrottler(emberEventName);
```

Comment:
this being a function seems uneeded, why not just inline `() => this.trigger(emberEventName);` ?

Reply:
Good call.